CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### BREAKING CHANGES
+- Remove `type` in `Divider` component @layershifter ([#558](https://github.com/stardust-ui/react/pull/558))
```

Comment:
Perhaps note that `type` is "replaced" by the `color` prop, instead of just removed?

Reply:
Updated, thanks 👍